sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java

```diff
@@ -254,17 +255,20 @@ public EventDirectorySteps(
     When(
         "I open the first event from events list",
         () -> webDriverHelpers.clickOnWebElementBySelector(FIRST_EVENT_ID_BUTTON));
+
     And(
         "I click Create Case for Event Participant",
         () -> webDriverHelpers.clickOnWebElementBySelector(CREATE_CASE_BUTTON));
+
     Then(
         "I check that number of displayed Event results is {int}",
         (Integer number) ->
             assertHelpers.assertWithPoll20Second(
                 () ->
                     Assert.assertEquals(
-                        webDriverHelpers.getNumberOfElements(CASE_GRID_RESULTS_ROWS),
```

Comment:
why this change? the method is used also in other tests and can cause failures

Reply:
I told Michal to do this, beacuse I had the same problem with the cases (assertion failure because despite 1 result in the grid it was finding 2) and as you suggested, I told Michal to change it with your directions - take the number from this button: 
![image](https://user-images.githubusercontent.com/96250118/153022017-cb91fdf1-af11-4460-b505-6f465b339942.png)

Reply:
I see, but in this case, if we touch any code used in other tests, we need to refactor how is used in other tests as well other we'll have a lot of failures.
cc: @MichaelSp